hu/hu_usb.cpp

```diff
@@ -429,7 +429,7 @@ int HUTransportStreamUSB::Start(byte ep_in_addr, byte ep_out_addr) {
         {
             logw("Wating for the device to reconnect");
             //Give it some time to reconnect
-            ms_sleep(1000);
+            ms_sleep(2000);
```

Comment:
I was thinking of introducing cmd line arguments and a way to pass settings to some of the functions, so these things can be changed without changing the code.